### PR TITLE
use metainfo folder instead of appdata

### PR DESCRIPTION
**Pull request by okias**
Lintian tag: appstream-metadata-in-legacy-location

AppStream metadata file was found in /usr/share/appdata/. The AppStream
XML files should be placed in /usr/share/metainfo/.

Refer to https://wiki.debian.org/AppStream/Guidelines for details.

Based-on work from: Pavel Rehak <pavel-rehak@email.cz>

Closes: https://github.com/transmission/transmission/pull/635
Closes: https://github.com/transmission/transmission/issues/1170
Fixes: 23d61b6457554203ea73951f3dd9f71a24be05ed

Signed-off-by: David Heidelberg <david@ixit.cz>

### CODE REVIEW

**Comment by okias** [comment]
Also I build & checked that `/usr/share/metainfo/transmission-gtk.metainfo.xml` is installed and contains everything expected.

@mikedld @ckerr if you manage merge this before creating 3.0.0 tarball it would nicely save maintainers some work to patch it with this!

Sorry for sending it late.

File: gtk/transmission-gtk.metainfo.xml.in [hunk 1]
```diff
@@ -12,7 +12,7 @@ Copyright 2017 Endless Mobile, Inc.
   <summary>Download and share files over BitTorrent</summary>
 
   <description>
-    <!-- Translators: these are the application description paragraphs in the AppData file. -->
+    <!-- Translators: these are the application description paragraphs in the MetaInfo file. -->
```
Comment (piotrdrag) [inline gtk/transmission-gtk.metainfo.xml.in:15]: You need to update `po/POTFILES.in` as well.

Comment (okias) [reply gtk/transmission-gtk.metainfo.xml.in:15]: thanks you!

**Comment by ckerr** [comment]
Just a heads-up in case you didn't see it, this PR is failing the Sanity check @ https://trtc.mikedld.com/viewLog.html?tab=buildLog&buildTypeId=Transmission_Sanity_GentooX8664&buildId=80936

**Comment by okias** [comment]
if I understand right, CI dont update potfiles after this commit.

**Comment by okias** [comment]
it seems after rebase without changes it now passes, probably some CI issue  @ckerr

**Comment by ckerr** [comment]
I know how long this PR has been in limbo and am going through the list to try and resolve older items.

@okias is this something you'd still like to see merged; and if so, would it be possible for you to update this PR to fix the merge conflicts?

**Comment by ckerr** [comment]
Superseded by #2624 so that I can fix merge conflicts. All credit and thanks goes to authors Pavel Rehak and @okias.

**Pull request closed without merging**